testing/src/cypress/page-objects/Institution-objects/ManageUsersObject.ts

```diff
@@ -35,12 +35,16 @@ export default class ManageUsersObject {
     return cy.get("[data-cy='addNewUserModal']");
   }
 
+  addNewUserModalTitle() {
+    return this.addNewUserModal().contains("Add new user");
+  }
+
   isAdminRadioButton() {
     return cy.get("[data-cy='isAdmin']");
   }
 
   isLegalSigningAuthorityButton() {
-    return cy.get("[data-cy='isLegalSigningAuthority]");
+    return cy.get("[data-cy='isLegalSigningAuthority']");
```

Comment:
In the future we can consider having a help method to stop writing `"[data-cy='something']"` every time, as below:
```ts
return cy.get(dataCY("isLegalSigningAuthority"));
```